devicetypes/smartthings/child-switch-health.src/child-switch-health.groovy

```diff
@@ -0,0 +1,50 @@
+/**
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition(name: "Child Switch Health", namespace: "smartthings", author: "SmartThings", mnmn: "SmartThings", vid: "generic-switch") {
+		capability "Switch"
+		capability "Actuator"
+		capability "Sensor"
+		capability "Health Check"
+	}
+
+	tiles(scale: 2) {
+		multiAttributeTile(name: "switch", width: 6, height: 4, canChangeIcon: false) {
+			tileAttribute("device.switch", key: "PRIMARY_CONTROL") {
+				attributeState "on", label: '${name}', action: "switch.off", icon: "st.switches.light.on", backgroundColor: "#00a0dc"
+				attributeState "off", label: '${name}', action: "switch.on", icon: "st.switches.light.off", backgroundColor: "#ffffff"
+			}
+		}
+
+		main "switch"
+		details(["switch"])
+	}
+}
+
+def installed() {
+	sendEvent(name: "checkInterval", value: hcIntervalMinutes * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
```

Comment:
Where is `hcIntervalMinutes` set?

Reply:
Nowhere, I just got distracted and forgot to finish it.  I also haven't tested this yet.

Reply:
Looks like a copy paste issue. I think it makes sense to have a default checkInterval set here but the real value should be set by the parent to whatever it wants.